msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -53,38 +78,154 @@ See the following video to learn how to embed Share to Teams button:
+| locale | `data-locale` | String | Optional | en-US | User preferred language. |
+| target | `data-target` | String | Optional | self | Specifies whether the link opens in the same window, new tab, or new window. |
+| buttonType | `data-button-type` | String | No | primaryShareInMeeting | Specifies the button background color: `primaryShareInMeeting` or `secondaryShareInMeeting`. |
+| buttonSize | `data-button-size` | String | No | NA | Button size in pixels. |
+| shareinmeeting | `data-allow-share-in-meeting` | String | No | NA | Button size in pixels. |
+
+## End user experience
+
+# [Chat or channel](#tab/chatchannel)
+
+To share the content to a chat or channel:
+
+1. Open the web app in the browser and select **Share to Teams**. The Share to Teams dialog opens.
+
+    :::image type="content" source="../../assets/images/share-to-teams/share-to-teams-browser.png" alt-text="Screenshot shows the Share to Teams button in the browser.":::
+
+1. Add the chat or channel name.
+
+    :::image type="content" source="~/assets/images/share-to-teams-popup.png" alt-text="Screenshot shows the Share to Teams dialog to add chat or channel name.":::
+
+   > [!NOTE]
+   > If the app isn't added in chat or channel scope, an app consent appears in the Share to Teams dialog.
+
+1. Select **Share**. The app gets added in the shared scope.
+
+# [Live meeting](#tab/live)
+
+To share content to an ongoing meeting:
+
+1. Open the web app in the browser and select **Share to Teams**. The Share to Teams dialog opens.
 
-1. If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force **Share** to render at the appropriate place in the pipeline.
+    :::image type="content" source="../../assets/images/share-to-teams/share-to-teams-browser.png" alt-text="Screenshot shows the Share to Teams button in the browser to share in meeting.":::
+
+1. Add the meeting name.
+
+    :::image type="content" source="~/assets/images/share-to-teams-popup.png" alt-text="Screenshot shows the  Share to Teams dialog to add the meeting name.":::
+
+   > [!NOTE]
+   > If the app isn't added in the meeting scope, an app consent appears in the Share to Teams dialog.
+
+1. Select **Share**. A consent dialog appears to share the content in the meeting.
+
+1. Select **Start sharing**.
+
+   :::image type="content" source="../../assets/images/share-to-teams/start-share-live-meeting.png" alt-text="Screenshot shows how to share apps in teams meeting.":::
+
+1. The web app is shared to meeting stage and all the participants can interact and work together.
+
+   :::image type="content" source="../../assets/images/share-to-teams/share-meeting-live.png" alt-text="Screenshot shows app shared to the teams meeting stage.":::
+
+# [Upcoming meeting](#tab/meetnow)
+
+If there's no ongoing meeting, the user can select **Meet Now** to begin a meeting. To open content in a scheduled call or to begin a new meeting from Share to Teams, follow the steps:
```

Comment:
This should be part of the earlier section - If there's no ongoing meeting, the user can select **Meet Now** to begin a meeting.